data/discovery/register/citizens-advice-bureau-membership-organisation-type.yaml

```diff
@@ -0,0 +1,9 @@
+fields:
+- citizens-advice-bureau-membership-organisation-type
+- name
+- start-date
+- end-date
+phase: discovery
+register: citizens-advice-bureau-membership-organisation-type
+registry: citizens-advice
```

Comment:
See above.

`citizens-advice` isn't in [ORJ](https://github.com/openregister/openregister-java/blob/master/src/main/resources/config/public-bodies.yaml) or even the [Whitehall API](https://www.gov.uk/api/organisations).  I think we should make this `cabinet-office` for now.

@michaelabenyohai how can we handle this?